docs/static/setup.sh

```diff
@@ -121,7 +121,8 @@ select opt in "${options[@]}" "Quit"; do
     18 ) shield_title="Tidbit" shield="tidbit"; split="n" break;;
     19 ) shield_title="Eek!" shield="eek"; split="n" break;;
     20 ) shield_title="BFO-9000" shield="bfo9000"; split="y"; break;;
-    21 ) shield_title="Helix" shield="helix"; split"y"; break;;
+    21 ) shield_title="Helix (5)" shield="helix"; split="y"; break;;
+    22 ) shield_title="Helix (4)" shield="helix_4"; split="y"; break;;
```

Comment:
I'm wondering if this should be listed as "Helix" and "Helix 4 Row"? It seems to me that this variant is much less popular?

Reply:
I'm not sure about which is more used, I only know one other person who uses one and he has both so 🤷‍♂️ 

Either way, how would you feel about merging this into one helix with multiple kscans and transforms? I'm not super familiar with devicetree syntax, so I'm not sure what config values would get set to differentiate between the 4 and 5 row variants of this board from a `zmk-config`.

Reply:
You're right here. It's probably better to have on helix with multiple transforms. I'm thinking just an additional transform might be enough. The Kyria shield has an example of this with the default transform and 5 column transform. Could you look into this?